### ISSUE TITLE
Bulleted and numbered list sometimes can't be added when an input box is empty

### ISSUE THREAD

**Issue by vhuseinova-msft** [microsoft/roosterjs#2417]
**Describe the bug**
Bulleted and numbered list sometimes can't be added after a user press Backspace button in the empty input box

**To Reproduce**
1. Open the demo website (https://microsoft.github.io/roosterjs/index.html)
2. **Without** typing anything in the input box, press Backspace button once
3. Click bulleted or numbered list button in the ribbon menu

**Expected behavior**
The bulleted or numbered list (dependending on the step 3 in the list above) should be added

Current behavior:
The list isn't added.

**Screenshots**

https://github.com/microsoft/roosterjs/assets/98852890/2aa09a09-cee2-4cc5-9745-bb85bd9f2efa

**Device Information**
 - OS: MscOS
 - Browser Edge
 - Version 121.0.2277.112 (Official build) (arm64)

**Additional context**

**Comment by vhuseinova-msft** [comment]
@JiuqingSong  Will this be fixed in the current RoosterJS packages? (as it looks like Content Model is still under active development)

**Comment by JiuqingSong** [comment]
No. the reason why we develop Content Model is just to fix this kind of bugs which is hard to fix in original editor.

We will release Content Model editor soon.
